### PR TITLE
Fix LineSegment3D : projection2d

### PR DESCRIPTION
**Pull request by WirajanDASILVA**
* **Please check if the PR fulfills these requirements**
- [X] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [X] Bug fix:
- [ ] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:

### CODE REVIEW

File: volmdlr/edges.py [hunk 1]
```diff
@@ -2798,8 +2798,10 @@ def point_distance(self, point):
         return distance, volmdlr.Point3D(*point)
 
     def plane_projection2d(self, center, x, y):
-        return LineSegment2D(self.start.plane_projection2d(center, x, y),
-                             self.end.plane_projection2d(center, x, y))
+        start, end = self.start.plane_projection2d(center, x, y), self.end.plane_projection2d(center, x, y)
+        if start != end:
+            return LineSegment2D(start, end)
+        return None
```
Comment (AxelRinghausen) [inline volmdlr/edges.py:2804]: Don't you want to return a single Point2D when `start == end` ?

**Review by AxelRinghausen — APPROVED** [review]

File: volmdlr/edges.py (hunk 1 above)
Comment (sbendjebla) [reply volmdlr/edges.py:2804]: Actually, if start == end > there will be a raise !

Comment (sbendjebla) [reply volmdlr/edges.py:2804]: Ah, sorry, you added this check (`start != end`), just to avoid the raise !

**Review by masfaraud — APPROVED** [review]
Ok good enough for now

**Pull request merged**